src/app/app.component.ts

```diff
@@ -56,28 +56,36 @@ export class AppComponent implements OnInit {
     private activatedRoute: ActivatedRoute,
     private router: Router
   ) {
+    this.initBasicServices();
+  }
+
+  private async initBasicServices() {
     // If loading the config earlier (in a module constructor or through APP_INITIALIZER) a runtime error occurs.
     // The EntityMapperService needs the SessionServiceProvider which needs the AppConfig to be set up.
     // If the EntityMapperService is requested to early (through DI), the AppConfig is not ready yet.
     // TODO fix this with https://github.com/Aam-Digital/ndb-core/issues/595
-    configService.loadConfig(entityMapper);
+    await this.configService.loadConfig(this.entityMapper);
+
+    this.analyticsService.init();
```

Comment:
Can we make sure to not use the analytics in development mode? I guess it would be quite misleading if all locally running apps are also sending analytics information. Angular has some ways to [check whether app is in prod or dev mode](https://stackoverflow.com/a/39535462/10713841)

Reply:
This is actually done here: https://github.com/Aam-Digital/ndb-core/blob/c17829bff957bf703662e172564f4d154c61be85/src/app/app.module.ts#L91-L93

Reply:
but we can save the setup as well, will add your suggested flag